README.md

```diff
@@ -362,6 +362,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 
 * [yuvadm/slingr](https://github.com/yuvadm/slingr) — A simple CLI for streaming media files over a local network to UPnP media renderers [<img src="https://api.travis-ci.org/yuvadm/slingr.svg?branch=master">](https://travis-ci.org/yuvadm/slingr)
 * [yuvadm/streamlib](https://github.com/streamlib/streamlib) — Play your favorite live video and audio streams from command line
+* [xiph/rav1e](https://github.com/xiph/rav1e) — The fastest and safest AV1 encoder. [<img src="https://travis-ci.org/xiph/rav1e.svg?branch=master">](https://travis-ci.org/xiph/rav1e)
```

Comment:
The travis image redirects to `https://api.travis-ci.org/xiph/rav1e.svg?branch=master` so if you could replace the url with that please?